test/specs/scale.time.tests.js

```diff
@@ -297,8 +297,8 @@ describe('Time scale tests', function() {
 		var xScale = chart.scales.xScale0;
 
 		// Counts down because the lines are drawn top to bottom
-		expect(xScale.ticks[0].value).toBe('Jan 2');
-		expect(xScale.ticks[1].value).toBe('May 8');
+		expect(xScale.ticks[0].value).toEqualOneOf(['Jan 1', 'Jan 2']);
```

Comment:
Would be better to not have approximative expectations. The failure might be because we use timestamps to create the moment object (line 284-285), we could try to use string instead (as other tests don't fail):

```javascript
parser: function(label) {
    return label === 'foo' ?
        moment('2000/01/02', 'YYYY/MM/DD') :
        moment('2016/05/08', 'YYYY/MM/DD');
}
```